Source/SwiftLintFramework/Rules/VerticalWhitespaceClosingBracesRule.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+import SourceKittenFramework
+
+private extension File {
+    func violatingRanges(for pattern: String) -> [NSRange] {
+        return match(pattern: pattern, excludingSyntaxKinds: SyntaxKind.commentAndStringKinds)
+    }
+}
+
+public struct VerticalWhitespaceClosingBracesRule {
+    public init() {}
+
+    private static let violatingToValidExamples: [String: String] = [
+        "    print(\"x is 5\")\n\n}": "    print(\"x is 5\")\n}",
```

Comment:
IMO this rule deserves a couple more complex examples since they're also used for unit tests. Particularly, I found additional complexity once I added cases where both opening and closing braces exist in the same line, such as in `map { foo }.filter {`

Reply:
It's also worth adding examples for closures with named parameters such as `map { foo in`

Reply:
Exactly. The examples should be extensive because they are used for the Unit Tests. Another detail I'd say it's missing is the location where the violations are triggered. The examples should show "↓" where the violation is triggered; the Unit Tests will make sure the violation is triggered on that place and the `Rules.md` file will show were the violation is expected to be displayed.

Reply:
hi @Dschee , I just noticed one example related to my comment above that wasn't covered by this rule. Do you think you can check this? This is the code snippet:
```swift
KingfisherManager.shared.retrieveImage(with: url, options: nil, progressBlock: nil) { image, _, _, _ in

  guard let img = image else { return }
```
(it has a blank line after the opening brace, but the opening brace line contains text after it)

Reply:
I just added these two examples and fixed the pattern accordingly. Thanks @gobetti!